filters/usa/california/west-hollywood/public-art-json-to-json.js

```diff
@@ -0,0 +1,35 @@
+const { response } = require("express");
```

Comment:
Can't have any node-related code in the file, as it will need to be run in the browser. Also, because this code must be able to be converted to text then to a function, the file should only contain the `const filter = function(data, stringify) {...}` and `return filter;`